rust/headless-client/src/lib.rs

```diff
@@ -50,6 +51,8 @@ pub mod windows;
 #[cfg(target_os = "windows")]
 pub(crate) use windows as platform;
 
+use ipc::{IpcServer, IpcStream};
```

Comment:
A pattern I sometimes use is to rename the inner items to just `Server` and `Stream` and then use `ipc::Server` and `ipc::Stream` to refer to it. That removes the need for these stuttering imports.